package.json

```diff
@@ -2,25 +2,27 @@
   "name": "react-vis",
   "version": "0.9.0",
   "license": "MIT",
-  "author": "Anton Bulyenov <antonb@uber.com>",
+  "author": "Visualization Team <visualization@uber.com>",
   "description": "Data visualization library based on React and d3.",
-  "keywords": [
-    "d3",
-    "react",
-    "visualization",
-    "chart",
-    "es6",
-    "babel"
-  ],
+  "main": "dist/index.js",
+  "files": ["dist", "main.css"],
   "repository": {
     "type": "git",
     "url": "https://github.com/uber-common/react-vis.git"
   },
   "bugs": {
     "url": "https://github.com/uber-common/react-vis/issues/new",
-    "email": "antonb@uber.com"
+    "email": "visualization@uber.com"
+  },
+  "scripts": {
+    "docs": "echo 'TODO'",
+    "start": "(cd examples && command -v yarn >/dev/null && yarn && npm start || npm install && npm start)",
+    "clean": "rm -rf dist && mkdir dist",
+    "build": "npm run clean && babel src -d dist --copy-files && node-sass src/main.scss main.css",
+    "lint": "eslint src tests examples --ignore-pattern node_modules",
+    "test": "tape -r babel-register tests/**/*.js",
+    "cover": "nyc --reporter=text --reporter=html --reporter=lcov npm test"
   },
```

Comment:
I see that the examples and `build-*` scripts were removed. Are they no longer needed?